docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -42,32 +42,38 @@ Currently ClickPipes does not support loading custom CA certificates.
 
   ![Fill out connection details](./images/cp_step2.png)
 
-4a. __Schema Registry__: A valid schema is required for Avro streams and optional for JSON. Specify the complete URL of your Schema Registry server along with the full path to the ID of the relevant schema document.  This schema will be used to parse ([AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent)) or validate (JSON) messages on the selected topic.  Avro messages that can not be parsed or JSON messages that fail validation will generate an error.  Note that ClickPipes will automatically retrieve an updated or different schema from the registry if indicated by the schema ID embedded in the message.
-5. Select your data format (we currently support a subset of ClickHouse formats). The UI will display a sample document from the selected source (Kafka topic, etc).
+5. Configure the schema registry. A valid schema is required for Avro streams and optional for JSON. This schema will be used to parse [AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent) or validate JSON messages on the selected topic.
+Avro messages that can not be parsed or JSON messages that fail validation will generate an error.  Note that ClickPipes will automatically retrieve an updated or different schema from the registry if indicated by the schema ID embedded in the message.
+There are two ways to format the URL path to retrieve the correct schema:
+- the path `/schemas/ids/[ID]` to the schema document by the numeric schema id. A complete url using a schema id would be `https://registry.example.com/schemas/ids/1000`
+- the path `/subjects/[subject_name]` to the schema document by subject name.  Optionally, a specific version can be referenced by appending `/versions/[version]` to the url (otherwise ClickPIpes
```

Comment:
```suggestion
- the path `/subjects/[subject_name]` to the schema document by subject name.  Optionally, a specific version can be referenced by appending `/versions/[version]` to the url (otherwise ClickPipes
```